client/my-sites/upgrades/checkout-thank-you/index.jsx

```diff
@@ -193,55 +198,50 @@ var CheckoutThankYou = React.createClass( {
 
 		return (
 			<div>
-				{ React.createElement( componentClass, {
-					selectedSite: selectedSite,
-					isFreeTrial: this.freeTrialWasPurchased(),
-					locale: i18n.getLocaleSlug(),
-					domain: domain
-				} ) }
+				<div className="checkout-thank-you__features-header">
+					{ this.translate( "Get started with your site's new features" ) }
+				</div>
+
+				<div className="checkout-thank-you__purchase-details-list">
+					{ React.createElement( componentClass, {
+						selectedSite: selectedSite,
+						isFreeTrial: this.freeTrialWasPurchased(),
+						domain: domain
```

Comment:
I agree but if we don't start somewhere we're going to implement the new `Thank You` page and it will still be old Javascript :).